x-pack/plugins/profiling/common/elasticsearch.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface ProfilingESEvent {
+  '@timestamp': string;
+  ContainerName: string;
+  ThreadName: string;
+  Count: number;
+  HostID: string;
+  PodName: string;
+  ProjectID: string;
+  StackTraceID: string;
+}
+
+export interface ProfilingStackTrace {
+  FrameID: string[];
+  LastSeen: number;
+  Type: number;
```

Comment:
`Type` should be an array with the same size as `FrameID` is. Each FrameID has its own Type.

Reply:
👍 Also I think `FrameID` is no longer an array of strings, but a single string, no? I need to update my locally running branch of prodfiler.

Reply:
> FrameID is no longer an array of strings

Just on the ES side (field in the index). After querying we have to parse and split the single value back into an array.
So if the interface resembles what we get from ES / mget, `FrameID` should be string.